deploy-board/deploy_board/static/js/components/capacitycomponents.js

```diff
@@ -266,29 +272,61 @@ Vue.component("static-capacity-config", {
     <form-danger v-show="showSizeError" :alert-text="sizeError"></form-danger>
     <form-warning v-show="showSizeWarning" :alert-text="sizeWarning"></form-warning>
     <form-warning v-show="showImbalanceWarning" :alert-text="imbalanceWarning"></form-warning>
+    <form-danger v-show="showTerminationError" :alert-text="terminationError"></form-danger>
     </div>`,
     props: {
         originalCapacity: Number,
         remainingCapacity: Number,
         placements: Object,
+        terminationLimit: Number,
+        groupName: String,
     },
     data: function() {
         return {
             capacity: this.originalCapacity,
+            terminationLimit: this.terminationLimit,
+            groupName: this.groupName,
             showSizeError: false,
             showImbalanceWarning: false,
             sizeError: '',
             imbalanceWarning: '',
             showSizeWarning: false,
             sizeWarning: '',
+            showTerminationError: false,
+            terminationError: '',
         }
     },
     methods: {
         onCapacityChange: function (value) {
+            console.log("capacity change triggered");
             this.capacity = Number(value);
             this.showSizeError = false;
             this.showSizeWarning = false;
+            console.log("debug");
+            this.showTerminationError = false;
+            this.terminatingHostCount = 0;
+            console.log("debug-before");
+            $.ajax({
+                type: 'GET',
+                url: `/groups/${this.groupName}/hosts`,
+                dataType: "json",
+                beforeSend: function(xhr, settings) {
+                    var csrftoken = getCookie('csrftoken');
+                    xhr.setRequestHeader("X-CSRFToken", csrftoken);
+                },
+                success: function (data) {
+                    console.log(data);
+                    this.terminatingHostCount = data.length;
+                    console.log(this.terminatingHostCount);
+                },
+                error: function (data) {
+                    console.log(data);
+                    this.terminatingHostCount = 0;
+                }
+            });
+            console.log("debug-mid");
```

Comment:
Can we please do this in `validateSize()` and only do it when `this.terminationLimit !== null` ?